pkg/federated/README.md

```diff
@@ -8,7 +8,11 @@ Use federated search to query and combine results from multiple Red Hat Advanced
     - Managed Hub clusters mu have RHACM 2.9.0 or later
 
 ## Setup
-Execute the script at `./setup.sh` to configure Global Search on the Global Hub cluster.  
+Execute the script at `./setup.sh` to configure Global Search on the Global Hub cluster. 
+
```

Comment:
Should we note that this script will need to be re-executed if a new cluster if imported after the setup?

Reply:
It is on the "Notes" below.
> You must re-run this script when a Managed Hub is added.
Maybe we could add a "Limitations" section to make sure we set the correct expectations. Thoughts?

Reply:
Oh, missed the notes. That should be fine.